triggers/service/BUILD.bazel

```diff
@@ -73,10 +73,8 @@ da_scala_binary(
 da_scala_test(
     name = "tests",
     srcs = glob(["src/test/scala/**/*.scala"]),
-    data = [
-        ":test-model.dar",
-        "@toxiproxy_nix//:bin/toxiproxy-cmd",
-    ],
+    data = [":test-model.dar"] + (["@toxiproxy_dev_env//:bin/toxiproxy-cmd"] if not is_windows else ["@toxiproxy_dev_env//:toxiproxy-server-windows-amd64.exe"]),
+    jvm_flags = ["-Dcom.daml.toxiproxy=$(rootpath @toxiproxy_dev_env//:bin/toxiproxy-cmd)"] if not is_windows else ["-Dcom.daml.toxiproxy=$(rootpath @toxiproxy_dev_env//:toxiproxy-server-windows-amd64.exe)"],
```

Comment:
I don’t think you actually need the condition here `$(rootpath @toxiproxy_dev_env//:toxiproxy)` should be sufficient. In Bazel that works fine but the runfiles libraries for Scala and others don’t seem to resolve file groups and aliases properly ime which is probably why just locating it in Scala didn’t work.

Reply:
we tried many variants of the above. in the end, what you see is the only thing we managed to get to work :)

Reply:
> I don’t think you actually need the condition here `$(rootpath @toxiproxy_dev_env//:toxiproxy)` should be sufficient.

Trouble is, on Linux this is a `filegroup` containing multiple files so Bazel won't accept `rootpath`. You can use `rootpaths` but then you need to pick the right file path out of those after the fact.

Reply:
ah makes sense. thanks for the explanation!